.travis.yml

```diff
@@ -97,22 +91,14 @@ before_install:
     - |
       # Install sigchild-enabled PHP to test the Process component on the lowest PHP matrix line
       if [[ ! $deps && $PHP = ${MIN_PHP%.*} && ! -d php-$MIN_PHP/sapi ]]; then
-          wget http://museum.php.net/php5/php-$MIN_PHP.tar.bz2 -O - | tar -xj &&
+          wget http://php.net/get/php-$MIN_PHP.tar.bz2/from/this/mirror -O - | tar -xj &&
           (cd php-$MIN_PHP && ./configure --enable-sigchild --enable-pcntl && make -j2)
       fi
 
     - |
-      # Install extra PHP extensions
-      if [[ ! $skip && $PHP = 5.* ]]; then
-          ([[ $deps ]] || tfold ext.symfony_debug 'cd src/Symfony/Component/Debug/Resources/ext && phpize && ./configure && make && echo extension = $(pwd)/modules/symfony_debug.so >> '"$INI") &&
-          tfold ext.apcu4 'echo yes | pecl install -f apcu-4.0.11'
-      elif [[ ! $skip && $PHP = 7.* ]]; then
-          tfold ext.apcu5 'echo yes | pecl install -f apcu-5.1.6'
-      fi
-
-    - |
       # Load fixtures
       if [[ ! $skip ]]; then
+          sleep 5
```

Comment:
Previously we did not require it because we had some extensions to compile, which gave some time for the ldap sever to start in the background. But now we have to sleep a bit to let it start.